cluster/src/main/java/com/hartwig/pipeline/transfer/SbpFileApiUpdate.java

```diff
@@ -30,14 +30,15 @@
     private final SbpRun sbpRun;
     private final Bucket sourceBucket;
     private final SbpRestApi sbpApi;
-    private final List<ApiFileOperation> fileOperations = new ArrayList<>();
+    private final Set<ApiFileOperation> fileOperations;
```

Comment:
As neither implementation of ApiFileOperation implement equals/hashcode this is only ensuring that the same reference isn't in the collection twice, which feels like masking another bug. I think we should keep going until we find where that is happening.

Still fine to keep idea but maybe then better we check for duplicates and log something before removing them?